src/mm-mapp.js

```diff
@@ -72,22 +72,40 @@ export class MmMapp extends HTMLElement {
   constructor() {
     // Always call super first in constructor
     super();
-    // SUPER IMPORTANT TO SET THIS UP FIRST SO THAT LEAFLET ISN'T WORKING WITH
-    // A HEIGHT=0 BOX BY DEFAULT.
-    this.style.display = "block";
+    
     let tmpl = document.createElement('template');
     tmpl.innerHTML =
     `<link rel="stylesheet" href="${new URL("leaflet.css", import.meta.url).href}">` +
     `<link rel="stylesheet" href="${new URL("leaflet.fullscreen.css", import.meta.url).href}">` +
     `<link rel="stylesheet" href="${new URL("mapml.css", import.meta.url).href}">`;
+    
     let shadowRoot = this.attachShadow({mode: 'open'});
     this._container = document.createElement('div');
-    // you have to include this otherwise you have to use quirks mode,
```

Comment:
Worried that this is gone

Reply:
This was setting the map element to a height of 100%, that's unlike any standard element, I believe it also required inheriting height from the body, which is probably not what authors would expect. It does require setting `html { height|min-height: 100%; }` though. Which I think is better, it's not unlike [the behavior of `<div>`s](https://stackoverflow.com/questions/7049875/why-doesnt-height-100-work-to-expand-divs-to-the-screen-height).

Reply:
> This was setting the map element to a height of 100%, that's unlike any standard element, I believe it also required inheriting height from the body, which is probably not what authors would expect.

Because the map element contains a div that contains a shadowRoot that in turn contains the Leaflet container div, I was trying to get the size of the Leaflet container div set to inherit the size of the map element which in the case of the example I was trying to set to the size of the user's screen.  It makes more sense to have a default size, but I think that still leaves us with using the size of the map element to set the size of the Leaflet container, and to make everything in between the Leaflet container and the map element be 100% of the size of its parent: 

map > div > shadowRoot > div.leaflet-container

Reply:
I believe this is the current situation, the shadowRoot host element has `display: contents` (i.e. it doesn't participate in box layout):

https://github.com/Maps4HTML/Web-Map-Custom-Element/blob/5f4ebde5b7d2e82fa8468687def65c35ac5b2de4/src/web-map.js#L104-L106